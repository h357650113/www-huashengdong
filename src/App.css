:root {
    --website-color-red: #eb5757;
    --website-color-blue: #7886ff;
    --website-color-gray-1: #e0e0e0;
    --website-color-gray-2: #313131;
    --website-color-gray-3: #a3a3a3;
    --website-color-orange-1: #f2c94c;
}

a {
    text-decoration: none;
}

@keyframes showItem {
    from {
        opacity: 0.3;
    }
    to {
        opacity: 1;
    }
}
