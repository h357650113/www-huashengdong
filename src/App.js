import { BrowserRouter, Routes, Route } from 'react-router-dom'
import './App.css'
import PageDefault from './Pages/Default/Default'
import PageAlbumItems from './Pages/AlbumItems/AlbumItems'
import PageDetail from './Pages/Detail/Detail'
import PageChart from './Pages/Chart/Chart'
import PageNodes from './Pages/Nodes/Nodes'

function App() {
    return (
        <BrowserRouter>
            <Routes>
                <Route path="/">
                    <Route index element={<PageDefault />} />
                </Route>
                <Route path="/item">
                    <Route path=":id/:title" element={<PageAlbumItems />} />
                </Route>
                <Route path="/detail">
                    <Route path=":id/:title" element={<PageDetail />} />
                </Route>
                <Route path="/nodes">
                    <Route index element={<PageNodes />} />
                </Route>
                <Route path="/chart" element={<PageChart />} />
            </Routes>
        </BrowserRouter>
    )
}

export default App
