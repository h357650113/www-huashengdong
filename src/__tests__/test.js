import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'

describe('AlbumItems', () => {
    test('renders correctly', () => {
        const component = renderer.create(<App />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
