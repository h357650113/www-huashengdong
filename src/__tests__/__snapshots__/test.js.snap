// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumItems renders correctly 1`] = `"loading..."`;
